CHANGELOG.rst

```diff
@@ -14,6 +14,23 @@ Unreleased
 
 *
 
+[7.0.0] - 2020-07-15
```

Comment:
Since you are designating this as backward incompatible, you should provide a note about that specifically calling out what is backward compatible and who it affects. There are examples in my draft OEP. 😉